### ISSUE TITLE
Constants protection breaks Newtonsoft JSON serialization

### ISSUE THREAD

**Issue by kryzys69** [mkaring/ConfuserEx#290]
Hi.
I've found that even simplest JSON serialization causes error when I use constants protection. I use:
- latest build of ConfuserEx 1.5
- target .NET framework 4.5.2
- latest Newtonsoft.Json from Nuget

```
using System;
using Newtonsoft.Json.Linq;

namespace ConfuserJsonTest
{
    class Program
    {
        private const string json = "{ \"a\":1, \"b\":2 }";
        static void Main(string[] args)
        {
            JObject jo = JObject.Parse(json);
            Console.Out.Write(jo.ToString());
            Console.ReadLine();
        }
    }
}
```

fragment of .crproj
```
<protection id="constants">
    <argument name="cfg" value="false" />
</protection>
```

I think it could be connected to #270

Please let me know if I can help in any way.

**Comment by mkaring** [comment]
@kryzys69: It would be very helpful if you could verify if the issue persists with the build you can download [here](https://ci.appveyor.com/project/mkaring/confuserex/builds/36978731/artifacts).

**Comment by kryzys69** [comment]
With the provided build this simple example works. But I use newer build ([640](https://ci.appveyor.com/project/mkaring/confuserex/builds/37102392)) and it doesn't work there. If it would be helpful, I can post a callstack.

**Comment by mkaring** [comment]
The pull-request for that issue has not been merged yet, due to the lack of feedback in #270 
I'll merge it later today, so it's present in the `master` branch.

**Comment by kryzys69** [comment]
Thanks for the info, I just saw this in issue #273. Thanks again for looking into this and hope it wasn't a problem
